service/storage_proxy.cc

```diff
@@ -5000,6 +5000,8 @@ result<::shared_ptr<abstract_read_executor>> storage_proxy::get_read_executor(lw
     if (cmd->allow_limit && _db.local().can_apply_per_partition_rate_limit(*schema, db::operation_type::read)) {
         auto r_rate_limit_info = choose_rate_limit_info(_db.local(), !is_read_non_local, db::operation_type::read, schema, token, trace_state);
         if (!r_rate_limit_info) {
+            slogger.debug("Read was rate limited");
```

Comment:
Log message below uses the term "Operation", maybe the same should be used here too.

Reply:
I used the same message as in the code path which handles the "replicas reject" case, here:

https://github.com/scylladb/scylladb/blob/114419d6ab42233f0bf05ac31e31f180c97ad5a5/service/storage_proxy.cc#L5095

Other, existing log messages around there already use "Read" instead of "Operation".

IMO it would make sense to change it the other way around, i.e. "Operation" -> "Read"/"Write". This would slightly increase the amount of information that those log messages provide. I can do this change in a followup.